src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -56,6 +56,7 @@ internal unsafe struct ManagedObjectWrapper
             public int UserDefinedCount;
             public ComInterfaceEntry* UserDefined;
             internal InternalComInterfaceDispatch* Dispatches;
+            public bool Destroyed;
```

Comment:
This field should not be needed. Checking for `Target == IntPtr.Zero` should work equally well.